activate/activate.nu

```diff
@@ -63,7 +63,7 @@ def activate_home_local [ user: string, host: string ] {
     let name = $"($user)" + (if ($host | is-empty) { "" } else { "@" + $host })
     log info $"Activating home configuration ($name) (ansi purple)locally(ansi reset)"
     log info $"(ansi blue_bold)>>>(ansi reset) home-manager switch --flake ($data.cleanFlake)#($name)"
-    home-manager switch -b backup --flake $"($data.cleanFlake)#($name)"
+    home-manager switch -b nixos-unified-.$(date now) --flake $"($data.cleanFlake)#($name)"
```

Comment:
Could you provide an example of the actual filename this creates both on macOS and Linux?

Also it would be good to document the backup behaviour in https://nixos-unified.org/activate.html#home

Reply:
The file name for `~/.bashrc` backup would look like
```
~/.bashrc.nixos-unified-$Tue, 14 Jan 2025 22:16:05 +0530 (now)
```

Reply:
On both macOS and Linux